core/src/main/java/binnie/core/models/DoublePassBakedModel.java

```diff
@@ -21,8 +21,8 @@
 	private static final int c = DefaultVertexFormats.ITEM.getColorOffset() / 4;
 	private static final int v = DefaultVertexFormats.ITEM.getNextOffset() / 4;
 	private final IBakedModel mainModel;
-	private int primaryColor = 0xFF000000;
```

Comment:
The constructor uses `|=` so this value is actually used, removing it will change the results.

Reply:
You right